content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,272 @@
+
+{{< text bash >}}
+$ INGRESS_POD=$(kubectl get pod -l istio=ingressgateway -n istio-system -o jsonpath="{.items[0].metadata.name}")
+$ INGRESS_POD_UID=$(kubectl get pods -n istio-system $INGRESS_POD -o jsonpath='{.metadata.uid}')
+{{< /text >}}
+
+2. Get the spire-server pod:
+
+{{< text bash >}}
+$ SPIRE_SERVER_POD=$(kubectl get pod -l app=spire-server -n spire -o jsonpath="{.items[0].metadata.name}")
+{{< /text >}}
+
+3. Attest the Spire Agent running on the node
+   {{< text bash >}}
+   $ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s_psat:cluster:demo-cluster \
+   -selector k8s_psat:agent_ns:spire \
+   -selector k8s_psat:agent_sa:spire-agent \
+   -node -socketPath /run/spire/sockets/server.sock
+   {{< /text >}}
+   
+   {{< text >}}
+   Entry ID         : d38c88d0-7d7a-4957-933c-361a0a3b039c
+   SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Parent ID        : spiffe://example.org/spire/server
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s_psat:agent_ns:spire
+   Selector         : k8s_psat:agent_sa:spire-agent
+   Selector         : k8s_psat:cluster:demo-cluster
+   {{</ text >}}
+
+4. and then register an entry for the pod:
+   {{< text bash >}}
+   $ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account \
+   -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s:sa:istio-ingressgateway-service-account \
+   -selector k8s:ns:istio-system \
+   -selector k8s:pod-uid:$INGRESS_POD_UID \
+   -dns $INGRESS_POD \
+   -dns istio-ingressgateway.istio-system.svc \
+   -socketPath /run/spire/sockets/server.sock
+   {{< /text >}}
+
+   {{< text >}}
+   Entry ID         : 6f2fe370-5261-4361-ac36-10aae8d91ff7
+   SPIFFE ID        : spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account
+   Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s:ns:istio-system
+   Selector         : k8s:pod-uid:63c2bbf5-a8b1-4b1f-ad64-f62ad2a69807
+   Selector         : k8s:sa:istio-ingressgateway-service-account
+   DNS name         : istio-ingressgateway.istio-system.svc
+   DNS name         : istio-ingressgateway-c48554dd6-cff5z
+   {{</ text >}}
+
+{{< warning >}}
+SPIFFE IDs for workloads must follow the Istio SPIFFE ID pattern: spiffe://<trust.domain>/ns/<namespace>/sa/<service-account>
+
+{{< /warning >}}
+
+See the [SPIRE help on Registering workloads](https://spiffe.io/docs/latest/deploying/registering/) to learn how to create new entries for workloads and get them attested using multiple selectors to strengthen attestation criteria.
+
+## Verifying Entries
+
+Confirm that identities were created for the workloads.

```

Comment:
Could you clarify to the user what we're doing here? Does this confirm that the whole integration is working or just that the registration for the Ingress Gateway is working correctly?